Errore Could not resolve "projects/design-angular-kit/src/public_api" in fase di compilazione progetto

### Esiste già una discussione sul tema che ti interessa, o su un tema simile?

- [X] Ho verificato e non esiste

### Versione della libreria

1.1.3

### Cosa

Durante il building dell'applicazione, una volta pushata su git, ricevo questo errore: 

✘ [ERROR] Could not resolve "projects/design-angular-kit/src/public_api"
    node_modules/design-angular-kit/fesm2022/design-angular-kit.mjs:16:22:
      16 │ import * as i1$5 from 'projects/design-angular-kit/src/public_api';
         ╵                       ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
  You can mark the path "projects/design-angular-kit/src/public_api" as external to exclude it from the bundle, which will remove this error and leave the unresolved path in the bundle.
✘ [ERROR] TS2307: Cannot find module 'projects/design-angular-kit/src/public_api' or its corresponding type declarations. [plugin angular-compiler]
    node_modules/design-angular-kit/lib/components/navigation/skiplink/skiplink-item/skiplink-item.component.d.ts:1:36:
      1 │ ...plinkComponent } from 'projects/design-angular-kit/src/public_api';

Non vorrei aver impostato io qualcosa di sbagliato ma fino a due giorni fa non ho avuto problemi e non ho fatto aggiornamenti particolari al progetto.

### Perché

Non vorrei fosse un bug presente in alcune versioni o comunque qualcuno potrebbe essersi trovato nella stessa situazione.

### Contesto

_No response_

### Altro

_No response_

--- a/projects/design-angular-kit/src/lib/components/form/transfer/transfer.component.ts
+++ b/projects/design-angular-kit/src/lib/components/form/transfer/transfer.component.ts
@@ -9,7 +9,7 @@ import { TransferStore } from './store/transfer.store';
 import { ItTransferListComponent } from './transfer-list/transfer-list.component';
 import { TransferItem } from './transfer.model';
 import { TranslateModule } from '@ngx-translate/core';
-import { ItIconComponent } from 'projects/design-angular-kit/src/public_api';
+import { ItIconComponent } from '../../utils/icon/icon.component';
 
 /**
  * Transfer

--- a/projects/design-angular-kit/src/lib/components/navigation/skiplink/skiplink-item/skiplink-item.component.ts
+++ b/projects/design-angular-kit/src/lib/components/navigation/skiplink/skiplink-item/skiplink-item.component.ts
@@ -2,7 +2,8 @@ import { ChangeDetectionStrategy, Component, Host, Input } from '@angular/core';
 import { NgTemplateOutlet } from '@angular/common';
 import { TranslateModule } from '@ngx-translate/core';
 import { inputToBoolean } from '../../../../utils/coercion';
-import { ItLinkComponent, ItSkiplinkComponent } from 'projects/design-angular-kit/src/public_api';
+import { ItLinkComponent } from '../../../core/link/link.component';
+import { ItSkiplinkComponent } from '../skiplink/skiplink.component';
 
 @Component({
   standalone: true,

--- a/projects/design-angular-kit/src/lib/components/navigation/skiplink/skiplink/skiplink.component.ts
+++ b/projects/design-angular-kit/src/lib/components/navigation/skiplink/skiplink/skiplink.component.ts
@@ -2,7 +2,7 @@ import { ChangeDetectionStrategy, Component, Input } from '@angular/core';
 import { NgTemplateOutlet } from '@angular/common';
 import { TranslateModule } from '@ngx-translate/core';
 import { inputToBoolean } from '../../../../utils/coercion';
-import { ItLinkComponent } from 'projects/design-angular-kit/src/public_api';
+import { ItLinkComponent } from '../../../core/link/link.component';
 
 @Component({
   standalone: true,

